compiler/optimizer/VPHandlers.cpp

```diff
@@ -2161,8 +2161,7 @@ TR::Node *constrainIaload(OMR::ValuePropagation *vp, TR::Node *node)
           && base->isNonNullObject()
           && base->getClassType()
           && base->getClassType()->asFixedClass()
-          && base->getClass()
-          && TR::Compiler->cls.isClassInitialized(vp->comp(), base->getClass()))
```

Comment:
My question remains as to why it is safe to remove this check - can you please comment with your justification?